phpBB/styles/prosilver/template/icon.html

```diff
@@ -0,0 +1,10 @@
+{% if TYPE == 'png' %}
+<img class="icon png-{{ ICON ~ S_CLASSES }}" src="{{ T_PATH }}"{% if TITLE %} alt="{{ TITLE }}"{% endif %} style="background: black;">
```

Comment:
whats with the hardcoded style?

Reply:
Oh yeah my bad, I downloaded the white png 😵 couldn't see it, like I said, the entire html file needs work as it's just a mock. Decided that that will be for you.
First want to make sure the PHP and loading times are correct.